password_cracker.md

```diff
@@ -302,6 +309,13 @@ When the worker threads finish a task, each thread will print the number of pass
 After all worker threads finish each task, the main thread will print the password (if found), the total number of hashes, the wall clock and CPU time spent on that task, and the ratio of CPU time to wall clock time.
 Note that we have not provided any of the timing print statements in `cracker2`.
 
+### Bounds
+1 <= thread_pool_size <= 13
```

Comment:
Is this here twice?

Reply:
Nope, added bounds for both version 1 and 2. Tests were significantly different.

Reply:
Combined into one now